docs/next.config.js

```diff
@@ -5,4 +5,149 @@ const withNextra = nextra({
   themeConfig: "./theme.config.tsx",
 });
 
-export default withNextra();
+export default withNextra({
+  async redirects() {
+    return [
+      {
+        source: "/what-is-mud",
+        destination: "/introduction",
+        permanent: true,
```

Comment:
imo while things are still in flux, would recommend using `permanent: false` 

browsers, search engines, CDNs, etc. heavily cache permanent redirects and they're hard to unwind

Reply:
I disagree, because I don't think we'll go back to the old docs, so the redirects will stay accurate until we redo the doc outline again, and hopefully that will be a very long time from now.

Reply:
I am not saying we'll go back to the old docs, but we basically kill any chance of using this URL in the future. With these redirects, we lose the ability in the future to have a page on `/store` for example.

Reply:
You convinced me. I'll change that.